src/routes/+layout.server.ts

```diff
@@ -1,6 +1,28 @@
 import type { LayoutServerLoad } from './$types';
+import * as cron from 'node-cron';
+
+let firstRun = true;
+
+export const load: LayoutServerLoad = async ({ locals, fetch }) => {
+	if (firstRun) {
+		console.log('FIRST RUN');
+		cron.schedule('*/1 * * * *', async function () {
+			console.log('*** CRON JOB START ***');
+			const res = await fetch('/reminder', {
```

Comment:
I think we shouldn't be fetching from an endpoint here. Stuff like the reminder functionality (and messaging itself) should go in lib/server and be invoked directly. That avoids a lot of issues and some overhead.